website/client/src/components/tasks/task.vue

```diff
@@ -1068,6 +1069,13 @@ export default {
         }
       }
 
+      if (firstDrops) {
+        Vue.set(user.items.eggs, firstDrops.egg, 0);
```

Comment:
Just one quick thing, here should we run Vue.set only if the user doesn’t have any of that egg/potion? Otherwise we reset it to zero even if they already bought one. Should be pretty rare at this stage but still possible

Reply:
We should only be running the first drops code if the user has never had any potions or eggs. But I can definitely put a safety check in, as I did elsewhere!